### PR TITLE
Add types for diffable-html

### PR DESCRIPTION
**Pull request by ZhangYiJiang**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [X] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [X] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [X] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [X] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [X] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [X] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

diffable-html GitHub page: https://github.com/algolia/diffable-html

The sortAttributes option is not documented in the README but it is in the source https://github.com/algolia/diffable-html/blob/master/index.js#L23

### CODE REVIEW

File: types/diffable-html/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// Type definitions for diffable-html 5.0
+// Project: https://github.com/rayrutjes/diffable-html#readme
+// Definitions by: Zhang Yi Jiang <https://github.com/DefinitelyTyped>
```
Comment (peterblazejewicz) [inline types/diffable-html/index.d.ts:3]: this one mascarades your GitHub handle with DT handle? What's the reason behind this?

Comment (ZhangYiJiang) [reply types/diffable-html/index.d.ts:3]: Weird, I'm not sure how that happened - this was generated by the `dts-gen` command and I didn't notice it did this. I'll fix it

File: types/diffable-html/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function format(
+    html: string,
+    options?: {
+        sortAttributes?: (attrs: string[]) => string[];
```
Comment (peterblazejewicz) [inline types/diffable-html/index.d.ts:9]:
```suggestion
        sortAttributes?: (names: string[]) => string[];
```
https://github.com/algolia/diffable-html/blob/master/index.js#L23
https://github.com/algolia/diffable-html/blob/master/__tests__/index.js#L275
The implementer has the last saying, but let's keep this aligned with JS

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@ZhangYiJiang thanks!

**Pull request merged**